internal/cmd/flags.go

```diff
@@ -71,6 +71,18 @@ var (
 	SetPflagPluginConfig = func(fs *pflag.FlagSet, p *[]string) {
 		fs.StringArrayVarP(p, PflagPluginConfig.Name, PflagPluginConfig.Shorthand, nil, PflagPluginConfig.Usage)
 	}
+
+	PflagUserMetadata = &pflag.Flag{
+		Name:      "user-metadata",
+		Shorthand: "m",
+	}
+
+	PflagUserMetadataSignUsage = "{key}={value} pairs that are added to the signature payload"
+	PflagUserMetadataVerifyUsage = "user defined {key}={value} pairs that must be present in the signature for successful verification if provided"
+
+	SetPflagUserMetadata = func(fs *pflag.FlagSet, p *[]string, usage string) {
+		fs.StringArrayVarP(p, PflagUserMetadata.Name, PflagUserMetadata.Shorthand, nil, usage)
+	}
```

Comment:
nit: Do you want to remove empty lines in between just to make it a code block and consistent?